lodash.js

```diff
@@ -2373,17 +2374,9 @@
      * @param {Object} prototype The object to inherit from.
      * @returns {Object} Returns the new object.
      */
-    var baseCreate = (function() {
-      function object() {}
-      return function(prototype) {
-        if (isObject(prototype)) {
-          object.prototype = prototype;
-          var result = new object;
-          object.prototype = undefined;
-        }
-        return result || {};
-      };
-    }());
+    function baseCreate(proto) {
+      return objectCreate(!isObject(proto) ? null : proto);
```

Comment:
I thought `null` would be the default, why `{}`? Are we going to allow `baseCreate(null)` when Object.create is supported, or ignore because non shimmable?